uiServices/src/testSuiteServices_test.go

```diff
@@ -0,0 +1,187 @@
+package services
+
+import (
+	"github.com/go-chi/chi"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+)
+
+const validTestSuite = `
+{
+  "name": "TestSuiteService",
+  "testStrategy": "SuiteBased",
+  "testCases": [
+    {
+      "name":"file1.xml",
+      "preThinkTime": 1000,
+      "postThinkTime": 2000,
+      "execWeight": "Infrequent"
+    },
+    {
+      "name":"file2.xml"
+    }
+  ]
+}
+`
+const TestSuiteMissingRequired = `
+{
+  "testCases": [
```

Comment:
add the name in so that `TestMissingRequiredTestSuitePost` is caught by` !validateTestSuiteJsonWithSchema(buf.Bytes()) in testSuiteServices.go`